subgraph/origin/generated/schema.ts

```diff
@@ -0,0 +1,89 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+
```

Comment:
Please rename the file to AnchorSchema and also change the imports in `AnchorMapping.ts`

Reply:
Done!